### PR TITLE
Update android_remocons for Indigo 

### PR DESCRIPTION
**Pull request by kentsommer**
Updated and tested android_remocons for use on Indigo.

### CODE REVIEW

File: common_tools/src/main/java/com/github/rosjava/android_remocons/common_tools/apps/RosAppActivity.java [hunk 1]
```diff
@@ -166,6 +166,7 @@ public void onCreate(Bundle savedInstanceState) {
 
         if (masterAppName == null) {
             // App name extra content key not present on intent; no remocon started the app, so we are standalone app
+            Log.e("We are running as standalone", ":(");
```
Comment (stonier) [inline common_tools/src/main/java/com/github/rosjava/android_remocons/common_tools/apps/RosAppActivity.java:169]: Should this be an error? I perhaps need to show you the standalone mechanism next week.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+The master branch is currently dependant on stable hydro branches of the other rosjava and android
+core repositories (too much movement there to depend on master). Also dependant on the master branch of
+android_apps.
+
+### Rosinstaller
+
+
+* Master: https://raw.github.com/rosjava/rosjava/master/android_apps.rosinstall
+* Hydro: https://raw.github.com/rosjava/rosjava/hydro/android_apps.rosinstall
+=======
```
Comment (stonier) [inline README.md:16]: We need to update this readme.

File: talker/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -19,7 +19,7 @@
             <category android:name="android.intent.category.LAUNCHER" />
         </intent-filter>
         <intent-filter>
-            <action android:name="com.github.rosjava.android_remocons.listener.TalkerActivity" />
+            <action android:name="com.github.rosjava.android_remocons.talker.TalkerActivity" />
             <category android:name="android.intent.category.DEFAULT" />
```
Comment (stonier) [inline talker/src/main/AndroidManifest.xml:23]: Ha, I am babo :)

File: common_tools/src/main/java/com/github/rosjava/android_remocons/common_tools/apps/RosAppActivity.java (hunk 1 above)
Comment (kentsommer) [inline common_tools/src/main/java/com/github/rosjava/android_remocons/common_tools/apps/RosAppActivity.java:169]: The log was just added as debug while I was troubleshooting and forgot to remove it. In this case, not being started by a remocon should probably result in an error though.

**Pull request merged**